guillotina/api/container.py

```diff
@@ -53,6 +55,40 @@ class DefaultGET(Service):
         return await serializer()
 
 
+async def create_container(parent: IDatabase, container_id: str,
+                           container_type: str='Container',
+                           owner_id: Optional[str]=None,
+                           emit_events: bool=True, **data):
+    container = await create_content(
+        container_type, id=container_id,
+        **data)
+
+    # Special case we don't want the parent pointer
+    container.__name__ = container_id
+
+    await parent.async_set(container_id, container)
+    await container.install()
+
+    if owner_id is None:
+        try:
+            owner_id = get_authenticated_user_id()
+        except RequestNotFound:
+            # fallback to root user as the owner
+            owner_id = 'root'
```

Comment:
Could we prevent assiging owner if there is no owner defined  on argss?

Reply:
yup, that is fine with me!